src/Tools/DynamoAddInGenerator/Program.cs

```diff
@@ -103,7 +113,7 @@ internal static void DeleteExistingAddins(IRevitProductCollection products)
         /// versions of Revit.
         /// </summary>
         /// <param name="products">A collection of revit installs.</param>
-        internal static void GenerateAddins(IRevitProductCollection products)
+        internal static void GenerateAddins(IRevitProductCollection products, string excludePath = null)
```

Comment:
Can you please also update the documentation for this method to describe excludePath.